python/pyspark/mllib/recommendation.py

```diff
@@ -224,7 +228,7 @@ class ALS:
     """
 
     @classmethod
-    def _prepare(cls, ratings):
+    def _prepare(cls, ratings: Any) -> RDD:
```

Comment:
Returned type should be properly annotated with `RDD[Rating]`.

We might also annotate `ratings` as `Union[RDD[???], DataFrame])`, but it is OK to have `Any` in this case.